packages/core/src/lib/switch.spec.ts

```diff
@@ -10,28 +10,17 @@ describe('Switch', () => {
     expect(screen.queryByRole('tooltip')).toBeNull();
   });
 
-  it('Renders with label and tooltip', async () => {
-    render(Switch, {
+  it('Renders with label and tooltip', () => {
+    const { container } = render(Switch, {
       label: 'Switch Label',
       tooltip: 'Tooltip Message',
     });
 
     const label = screen.getByText('Switch Label');
     expect(label).toBeVisible();
 
-    const tooltipIcon = screen.getByRole('button', {
-      name: 'information-outline icon',
-    });
-    expect(tooltipIcon).toBeVisible();
-
-    await fireEvent.mouseEnter(tooltipIcon);
-
-    await waitFor(() => {
-      expect(screen.getByText('Tooltip Message')).toBeVisible();
-    });
-
-    const icon = screen.getByRole('img', { name: /information-outline/u });
-    expect(icon).toBeVisible();
+    const icon = container.querySelector('svg') as Element;
```

Comment:
[q] - is the reason we are no longer testing the tooltip's behavior on hover in the switch tests because that should already be tested in the tooltip specific tests?

Reply:
yeah, two reasons:
* IMO this test was overstepping its boundary
* Mocking hover behavior was being flakey once I changed the selector, and I don't think unit tests should really attempt to simulate hover states, I think if we want to test this we should instead test that props are passed correctly to the tooltip